src/lazy/encoder/binary/v1_1/flex_sym.rs

```diff
@@ -95,7 +95,7 @@ impl<'top> FlexSym<'top> {
             Ordering::Equal => {
                 // Get the first byte following the leading FlexInt
                 let flex_int_len = value.size_in_bytes();
-                if input.len() < flex_int_len {
+                if input.len() <= flex_int_len {
```

Comment:
🪧 This bug surfaced while I was working on an example file from @jobarr-amzn. This fixed it.